docs/src/api/params.md

```diff
@@ -213,6 +213,14 @@ Whether to ignore HTTPS errors during navigation. Defaults to `false`.
 
 Toggles bypassing page's Content-Security-Policy.
 
+## context-option-baseURL
+- `baseURL` <[string]>
+
+Prefixes the URL on navigations when a path (no origin) is passed to [`method: Page.goto`]. You can specify also a
+base URL with a path. Examples:
+* baseURL: `http://localhost:3000` and navigating to `/bar.html` results in `http://localhost:3000/bar.html`
+* baseURL: `http://localhost:3000/foo/` and navigating to `/bar.html` results in `http://localhost:3000/foo/bar.html`
```

Comment:
Is this example correct now? Shouldn't it say `./bar.html`?

Reply:
with the new implementation you are absolutely right! 👍